Gemfile

```diff
@@ -113,6 +113,7 @@ group :test do
   gem "minitest-bisect"
   gem "minitest-retry"
   gem "minitest-reporters"
+  gem "rails-dom-testing", github: "rails/rails-dom-testing", branch: "master"
```

Comment:
I suspect we _don't_ want to depend on the GitHub branch itself. I wanted to open this ahead of bumping the rails-dom-testing version just to see what the upgrade would require.

Additionally, I'm not sure how we'd upgrade this without adding this line to `Gemfile`. Would we upgrade the gemspec's dependencies ([actionpack](https://github.com/rails/rails/blob/ab8e3d22cbac85f1e5742dd13320cda6bd54e191/actionpack/actionpack.gemspec#L40), [actionview](https://github.com/rails/rails/blob/ab8e3d22cbac85f1e5742dd13320cda6bd54e191/actionview/actionview.gemspec#L40)) to `~> 3.0`?